src/components/Modal_Components/BackContinue.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
```

Comment:
This file is showing as if it's brand new but it's just a side effect of fixing the import casing.